src/AppBundle/Form/PostType.php

```diff
@@ -43,13 +43,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'attr' => array('autofocus' => true),
                 'label' => 'label.title',
             ))
-            ->add('summary', 'textarea', array('label' => 'label.summary'))
-            ->add('content', 'textarea', array(
+            ->add('summary', 'Symfony\Component\Form\Extension\Core\Type\TextareaType', array('label' => 'label.summary'))
```

Comment:
@bocharsky-bw then, when users to type a text with 256 characters or more will be truncated and this could be an unexpected result. I suggest then use `max_length` 255 for this widget.